style.css

```diff
@@ -0,0 +1,27 @@
+body {
+    background-color: #fff;
+    color: #333;
+  }
+.dark-mode {
```

Comment:
create  a single file for dark mode and track all changes in buttons and texts and make them more visble